test/powershell/Language/Scripting/ErrorPosition.Tests.ps1

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
```

Comment:
Why we need new file? 
It seems we can use existing.

Reply:
I used a new file because I couldn't find an existing test file that this category of issues can fall in.
Here we are not testing indexing into a null array, but the scenario where it fails to evaluate the switch statement's condition (it could fail for a different reason, and I just happen to use `$null[0]`).

As for the test you referenced, it looks more fit in `Language/Scripting` instead of `Language/Parser` because the error that is tested is a runtime error instead of a parsing one.

Reply:
Yes, I don't found a file with runtime error tests too. And I wonder. If we need better code coverage perhaps we should rename the file from `ErrorPosition` to `RuntimeErrors`.

Reply:
Runtime errors usually are tested along with the feature that they are associated with, no need for a separate test file only for validating runtime errors.
Error position is a feature used for reporting errors, so I think it's better to have a test file of its own category.